modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -4070,6 +4070,27 @@ CV_EXPORTS_W void approxPolyDP( InputArray curve,
                                 OutputArray approxCurve,
                                 double epsilon, bool closed );
 
+/** @brief Approximates a polygon with a convex hull with a specified accuracy and number of sides.
+
+The cv::approxBoundingPoly function approximates a polygon with a convex hull
+so that the difference between the contour area of the original contour and the new polygon is minimal.
+It uses a greedy algorithm for contracting two vertices into one in such a way that the additional area is minimal.
+Straight lines formed by each edge of the convex contour are drawn and the areas of the resulting triangles are considered.
+Each vertex will lie either on the original contour or outside it
+
+
+@param curve Input vector of a 2D point stored in std::vector or Mat.
```

Comment:
@param curve Input vector of a 2D points stored in std::vector or Mat, points must be float or integer.